### PR TITLE
style(hydration errors): consistent before/after style in modal

### LINKED ISSUE #77210: [Hydration Errors] Update Modal so all references to "Before" and "After" are bold, red&green, not say "Before Hydration"

### PR DESCRIPTION
**Pull request by michellewzhang**
closes https://github.com/getsentry/sentry/issues/77210

https://github.com/user-attachments/assets/b3b7c0ad-068f-4155-b82e-6bddec0fbd57

### CODE REVIEW

File: static/app/components/replays/diff/replaySideBySideImageDiff.tsx
Comment (ryan953) [inline static/app/components/replays/diff/replaySideBySideImageDiff.tsx:1]: One other thing I think would be helpful is to throw some borders around the children of `<ReplayGrid>`. Then the images that are side-by-side will have a border between them.

**Review by ryan953 — APPROVED** [review]

File: static/app/components/replays/diff/replaySideBySideImageDiff.tsx
Comment (michellewzhang) [reply static/app/components/replays/diff/replaySideBySideImageDiff.tsx:1]: like a green / red border?

Comment (ryan953) [reply static/app/components/replays/diff/replaySideBySideImageDiff.tsx:1]: Ya

Comment (michellewzhang) [reply static/app/components/replays/diff/replaySideBySideImageDiff.tsx:1]: yes you made a [ticket](https://github.com/getsentry/sentry/issues/77209) for it! i'll do it in a followup pr

Comment (ryan953) [reply static/app/components/replays/diff/replaySideBySideImageDiff.tsx:1]: goldfish memory

**Review by billyvg — COMMENTED** [review]
Should these all styled components instead of using inline `css`/`style` prop?

**Comment by michellewzhang** [comment]
> Should these all styled components instead of using inline `css`/`style` prop?

i can clean this up in a followup @billyvg

**Pull request merged**